sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/models/TableErrorCode.java

```diff
@@ -8,43 +8,277 @@
 /**
  * Defines values for error codes returned from the Tables service.
  */
-public class TableErrorCode extends ExpandableStringEnum<TableErrorCode> {
+public final class TableErrorCode extends ExpandableStringEnum<TableErrorCode> {
+    /**
+     * Static value {@code InvalidTableName}.
+     */
+    public static final TableErrorCode INVALID_TABLE_NAME = fromString("InvalidTableName");
+
+    /**
+     * Static value {@code InvalidPkOrRkName}.
+     */
+    public static final TableErrorCode INVALID_PK_OR_RK_NAME = fromString("InvalidPkOrRkName");
```

Comment:
Why do these two not have the "Value" suffix?

Reply:
None should have the value suffix, it was a mistake when copying the new entries from .NET

Reply:
Good catch!